tests/test-war-smoke/src/test/java/com/google/cloud/runtimes/jetty/test/smoke/JspIntegrationTest.java

```diff
@@ -14,37 +14,41 @@
  * limitations under the License.
  */
 
-package com.google.cloud.runtimes.jetty.tests;
+package com.google.cloud.runtimes.jetty.test.smoke;
 
 import static org.hamcrest.CoreMatchers.containsString;
 import static org.hamcrest.CoreMatchers.is;
 import static org.hamcrest.MatcherAssert.assertThat;
 
-import com.google.cloud.runtime.jetty.testing.HttpUrlUtil;
+import com.google.cloud.runtime.jetty.test.AbstractIntegrationTest;
+import com.google.cloud.runtime.jetty.test.ContainerRunner;
+import com.google.cloud.runtime.jetty.test.annotation.Local;
+import com.google.cloud.runtime.jetty.test.annotation.Remote;
+import com.google.cloud.runtime.jetty.util.HttpUrlUtil;
 
-import org.apache.commons.io.FileUtils;
-import org.apache.commons.io.IOUtils;
-import org.apache.commons.io.LineIterator;
 import org.junit.Assert;
+import org.junit.Test;
+import org.junit.runner.RunWith;
 
 import java.io.IOException;
-import java.io.StringReader;
 import java.net.HttpURLConnection;
 import java.net.URI;
 import java.util.Properties;
-import java.util.StringTokenizer;
 
-public abstract class AbstractJspIntegrationTest {
-
-  public abstract void testJspEnvironment() throws IOException;
+@RunWith(ContainerRunner.class)
```

Comment:
Can we move the `@RunWith` to `AbstractIntegrationTest`, so it doesn't have to be repeated in every test class?